src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -459,6 +466,40 @@ def test_sql_db_vcore_mgmt(self, resource_group, resource_group_location, server
                      JMESPathCheck('sku.capacity', vcore_capacity_updated),
                      JMESPathCheck('sku.family', vcore_family_updated)])
 
+    @ResourceGroupPreparer(name_prefix='clitest', location='eastus2')
```

Comment:
Can you change your name_prefix to something related to your service? e.g. clitest_sql_

Reply:
Fixed.